draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -313,14 +305,14 @@ The file header is followed by a series of data blocks.
 
 A block consists of a block header, containing various tables of common data,
 and some statistics for the traffic received over the block. The block header
-is then followed by a list of the Q/R pairs detailing the queries and responses
-received during the block. The list of Q/R pairs is in turn followed by a list
+is then followed by a list of the Q/R date items detailing the queries and responses
+received during the block. The list of Q/R data items is in turn followed by a list
```

Comment:
new nit: 'during the block'   Shouldn't we say something like 'during the processing of the block input' or